job/models.py

```diff
@@ -3956,17 +3956,296 @@ class Message(BaseModel, models.Model):
+            MaxValueValidator(1)
+        ],
+        null=True,
+        blank=True,
+        help_text=("Annual electric chiller energy consumption scalar as a fraction of total electric load applied to every time step"
+                "used to scale simulated default electric chiller load profile for the site's climate zone"
+        )
+    )
+
+    monthly_fractions_of_electric_load = ArrayField(
+        models.FloatField(
+            validators=[
+                MinValueValidator(0),
+                MaxValueValidator(1)
+            ],
+            blank=True
+        ),
+        default=list, blank=True,
+        help_text=("Monthly fraction of site's total electric consumption used up by electric chiller, applied to every hour of each month,"
+                    "to scale simulated default building load profile for the site's climate zone")
+    )
+
+    per_time_step_fractions_of_electric_load = ArrayField(
+        models.FloatField(
+            blank=True
+        ),
+        default=list,
+        blank=True,
+        help_text=("Per timestep fraction of site's total electric consumption used up by electric chiller."
+                    "Must be hourly (8,760 samples), 30 minute (17,520 samples), or 15 minute (35,040 samples)."
+                   )
+    )		
+
+    def clean(self):
+        error_messages = {}
+
+        # possible sets for defining load profile
+        if not at_least_one_set(self.dict, self.possible_sets):
+            error_messages["required inputs"] = \
+                "Must provide at least one set of valid inputs from {}.".format(self.possible_sets)
+
+        if len(self.blended_doe_reference_names) > 0 and self.doe_reference_name == "":
+            if len(self.blended_doe_reference_names) != len(self.blended_doe_reference_percents):
+                error_messages["blended_doe_reference_names"] = \
+                    "The number of blended_doe_reference_names must equal the number of blended_doe_reference_percents."
+            if not math.isclose(sum(self.blended_doe_reference_percents),  1.0):
+                error_messages["blended_doe_reference_percents"] = "Sum must = 1.0."
+
+        if self.doe_reference_name != "" or \
+                len(self.blended_doe_reference_names) > 0:
+            self.year = 2017  # the validator provides an "info" message regarding this)
+        
+        if len(self.monthly_fractions_of_electric_load) > 0:
+            if len(self.monthly_fractions_of_electric_load) != 12:
+                error_messages["monthly_fractions_of_electric_load"] = \
+                    "Provided cooling monthly_fractions_of_electric_load array does not have 12 values."
+        
+        # Require 12 values if monthly_tonhours is provided.
+        if 12 > len(self.monthly_tonhour) > 0:
+            error_messages["required inputs"] = \
+                "Must provide 12 elements as inputs to monthly_tonhour. Received {}.".format(self.monthly_tonhour)
```

Comment:
@zolanaj Thoughts on needing this validation if it's also done in Julia?

Reply:
(the same comment would apply to the current validation in electric and heating loads) I believe all of these validations were before we could pass back the errors from Julia. I'm now thinking that when we merge the in-process "error passing from Julia" PRs, we do a scrub of unnecessary/duplicate validation. So no action needed!

Reply:
Thanks @Bill-Becker!  I added issue https://github.com/NREL/REopt_API/issues/387 to note this as a future development effort and will merge this now.